less/forms.less

```diff
@@ -54,6 +54,167 @@
   }
 }
 
+.braveryPanelContainer {
```

Comment:
If i make my window as small as possible, the bravery panel is partially hidden because there's no vertical scrollbar